versions.yaml

```diff
@@ -7,7 +7,7 @@ modules:
     pxcentralOnpremUiLhbackend: "docker.io/portworx/pxcentral-onprem-ui-lhbackend-private:2.9.0-dev"
     pxcentralOnpremPostSetup: "docker.io/portworx/pxcentral-onprem-post-setup-base:2.9.0-dev"
     pxcentralOnpremPreSetup: "docker.io/portworx/pxcentral-onprem-hook-base:2.9.0-dev"
-    postgresql: "docker.io/portworx/postgresql:17.4.0-debian-12-r15"
+    postgresql: "docker.io/portworx/postgresql:17.4.0-debian-12-r19"
```

Comment:
In addition to this , we need to change in the values.yaml file (charts/px-central/values.yaml) which is the source for the image tags